clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -753,6 +753,9 @@ def err_drv_hlsl_bad_shader_required_in_target : Error<
 
 def err_drv_hlsl_bad_shader_unsupported : Error<
   "%select{shader model|Vulkan environment|shader stage}0 '%1' in target '%2' is invalid for HLSL code generation">;
+def err_drv_hlsl_dxc_bad_argument_value : Error<
```

Comment:
Can we use `err_drv_invalid_value` instead? I don't think there is any reason the error needs to say `dxc` in the text.